src/Link.js

```diff
@@ -1,7 +1,9 @@
 import styled from 'styled-components'
+import { colors } from './theme'
 
 const Link = styled.a`
   text-decoration: none;
+  color: ${colors['blue']};
```

Comment:
Instead of importing the theme module, you should use the theme context. This line would be written as `color: ${props => props.theme.colors.blue};`

You can read more about theming in the docs: https://www.styled-components.com/docs/advanced#theming

Reply:
Oh yeah, theme's colors are available from theme provider.

Updated to the blue from theme provider with a conditional CSS rule.
```
${props => props.theme ? `color: ${props.theme.colors.blue};` : null}
```

The thought is that if client opts out or forgets to use theme provider, then there wouldn't be any odd CSS rule key with empty string value such as: `color:` in the browser inspection. Plz review again at your next convenience.